packages/a11y-testing/src/definitions/Button/buttonBehaviorDefinition.ts

```diff
@@ -27,8 +27,9 @@ export const buttonBehaviorDefinition: Rule[] = [
     .forProps({ disabled: true, loading: true })
     .doesNotHaveAttribute('disabled'),
   BehaviorRule.root()
-    .forProps({ disabled: true })
-    .hasAttribute('aria-disabled', 'true'),
+    .forProps({ disabledFocusable: true })
+    .hasAttribute('aria-disabled', 'true')
```

Comment:
If we test both conditions at once, like add attribute and doesn't add, then documentation contains only one condition:

![image](https://user-images.githubusercontent.com/22620150/104914341-481d3080-598f-11eb-8995-1a2b009fe7fe.png)

Maybe we can split it into two:
`BehaviorRule.root()
    .forProps({ disabledFocusable: true })
    .hasAttribute('aria-disabled', 'true'),
  BehaviorRule.root()
    .forProps({ disabledFocusable: true })
    .doesNotHaveAttribute('disabled'),`
    
    and then we will get:    
![image](https://user-images.githubusercontent.com/22620150/104914568-9f230580-598f-11eb-9424-e3e74a4224ef.png)

or we need to go into documentation logic and do change there it will properly reflect both conditions used in one scenario.